articleview.cc

```diff
@@ -394,6 +402,16 @@ void ArticleView::showDefinition( Config::InputPhrase const & phrase, unsigned g
   currentWord = phrase.phrase.trimmed();
   if( currentWord.isEmpty() )
     return;
+
+  // search lightui+darkmode to jump to another section of this hack
+  // set the underlying Frame's background to Dark so that when
+  // the page is loading there is no flash when user has light ui + dark reader mode
+  // TODO: this is not efficient. rewrite ArticleView.ui so that the background of the search widget and the page is decoupled.
+  if( cfg.preferences.darkReaderMode ) {
+    setStyleSheet( "ArticleView {background-color: black;}" );
+  }
```

Comment:
can be moved into constructor of ArticleView

Reply:
Same as above.